### ISSUE TITLE
[Feature Request] Adhere to WAI-ARIA practices for menu navigation in speed dial

### ISSUE THREAD

**Issue by rudolfbyker** [vuetifyjs/vuetify#11437]
### Problem to solve
I'm working on keyboard navigation for accessibility. Currently, you can tab to a speed dial fab, and you can open/close the dials using spacebar, but you can't select a dial with the keyboard.

Relevant reads:
- https://www.w3.org/TR/wai-aria-practices/#menu
- https://webaim.org/techniques/keyboard/

### Proposed solution
After pressing space, the user should be able to navigate up and down with the keyboard arrows, just like when using VMenu. (Or left and right if the dials are horizontal.)

**Comment by johnleider** [comment]
In the latest version of Speed Dial it's able to navigate using tab / shift + tab. Is this sufficient?

**Comment by rudolfbyker** [comment]
:) I'm no longer using SpeedDial (that was 4 years ago), but your solution sounds accessible enough.
